sdk/ml/azure-ai-ml/tests/automl_job/unittests/test_automl_image_schema.py

```diff
@@ -121,18 +118,6 @@ def expected_image_target_column_name() -> str:
     return "label"
 
 
-@pytest.fixture
-def expected_image_sweep_settings() -> RestImageSweepSettings:
-    return RestImageSweepSettings(
-        limits=RestImageSweepLimitSettings(),
```

Comment:
I think we only need to remove the restimagesweeplimitsettings here?

Reply:
fixed